config/config.go

```diff
@@ -135,6 +137,16 @@ type OpenTracingReporter struct {
 	LocalAgentHostPort  string        `toml:"local-agent-host-port" json:"local-agent-host-port"`
 }
 
+// ProxyProtocol is the PROXY protocol section of the config.
+type ProxyProtocol struct {
+	// PROXY protocol acceptable client networks.
+	// Empty string means disable PROXY protocol,
+	// * means all networks.
+	Networks string `json:"networks" toml:"networks"`
```

Comment:
Copy this comment into the config file so that others can know the meanings of the configuration without taking a look at the code.